tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1657,3 +1657,37 @@ TEST(onnx, autoLoadInputs) {
   EXPECT_EQ(inputs.size(), 1);
   EXPECT_TRUE(inputTensor.getType().isEqual(inputs[inputName]->getType()));
 }
+
+TEST(onnx, shape) {
```

Comment:
Does this test pass without your fix?

Reply:
It doesn't. It has a pattern `Shape(Shape(X))` and it fails to get output of `Shape(X)`.